### DISCUSSION TITLE
TimeZoneInfo field serialized differently on Windows vs Linux agent in Azure

### DISCUSSION THREAD

**Discussion by verynewuser** [VerifyTests/Verify#809] [General]
I have a class having a property of `TimeZoneInfo` type. Calling `await Verifier.Verify(obj)` on it generates a *.Received.txt file, in which the value for this property is different when I test it on my development environment as compared to what it evaluates on Azure DevOps. My development environment is Windows and the Azure agent is Linux. On my local the object is serealized as 
```json
{
 timeZone: AUS Eastern Standard Time
}
```
whereas on Azure it is serialized as 
```json
{
 timeZone: Australia/Sydney
}
```

This behavior causes the test to fail in build pipeline. I have a temporary workaround which is to call `.ScrubMembersWithType<TimeZoneInfo>()` in a fluent-style, but is there a better way to tackle this?

**Comment by SimonCropp** [comment]
can u try v 19.11.0

**Reply by verynewuser** [reply]
I tried today but it still fails, though now the resulting string on Azure is different than it was before. It is the below now:
```json
{
  timeZone: Etc/UTC
}
```
